docs-user/async-posix-signal-control.md

```diff
@@ -0,0 +1,102 @@
+
+# Async POSIX Signal Control
+
+The Firefox profiler supports the use of asynchronous POSIX signals for a
+limited number of operations, specifically starting and stopping the profiler.
+This can be useful in situations where the normal UI or keyboard shortcut
+control of the profiler is not available.
+
+In brief, a user can send the POSIX signal `SIGUSR1` to the main Firefox process
+to start the profiler, and the signal `SIGUSR2` to stop the profiler. Upon
+recieiving the "stop" signal, the profiler will collect the current profile data
+and write it to a `.json` file in the user's downloads directory. For example,
+on MacOS:
+
+    // Send signal USR1 to the first (main) process of Firefox.app to start the profiler
+    kill -s USR1 `pgrep -f Firefox.app | head -n 1`
+    // wait for firefox to gather some data
+    sleep 10
+    // Send signal USR2 to the same process to stop the profiler
+    kill -s USR2 `pgrep -f Firefox.app | head -n 1`
+    // Find the result in the download folder
+    ls ~/Downloads/ | grep "profile.*json"
+
+From there, the user can open the written profile using the standard profiler
+UI, by navigating to [profiler.firefox.com](profiler.firefox.com), and selecting
```

Comment:
This link looks broken. Could you add `https://` as well here to make it an absolute url?